### DISCUSSION TITLE
Welcome to inno-doctor Discussions!

### DISCUSSION THREAD

**Discussion by imRohitSahu** [imRohitSahu/inno-doctor#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our team. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a team we
  build together 💪.

**Comment by eseymenler** [comment]
Hi my friend, firstly i like your project, because i have a problem with primary key and foreignkey relations, so i learned lots of things from you. And i wanna give automatic patient id, how can i do this. Could you give me hints or direct code? Thanks my friend.
